<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton {
       background-color: rgb(230, 230, 230);
       color: black;                         
       font-weight: bold;              
    }
    QPushButton:disabled {                  
       background-color: rgb(211, 215, 207);
       color: gray;                 
       font-weight: normal;
    }</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton {
       background-color: rgb(230, 230, 230);
       color: black;                         
       font-weight: bold;              
    }
    QPushButton:disabled {                  
       background-color: rgb(211, 215, 207);
       color: gray;                 
       font-weight: normal;
    }</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>59</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="videoLayout">
    <item>
     <widget class="QLabel" name="videoLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
